src/Adapter/Kpi/AverageResponseTimeKpi.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
```

Comment:
Could you update your license block with this URL please `https://devdocs.prestashop-project.org/` ? (here and the other files)

Reply:
we shouldn't update licence headers manually. We have a command for that reason. https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Command/UpdateLicensesCommand.php.
It should be used in dedicated PR to update the whole project headers when needed.

Reply:
I don't think the command is able to replace the existing headers, only add the missing ones So if we can anticipate the correct header it's better, but they'll all have to be updated anyway so I don't think this should be blocking for this PR (unless you have some other things to fix and you can use the occasion to do so)

Reply:
> I don't think the command is able to replace the existing headers
Yes it does replace them and its definetelly not something Marius should do manually now :smile: (i just tried the command, it updates the existing ones too).
Btw I also don't know when or where are the correct headers actually. Marius just copied those from other files (thousand of files having the same headers). I don't think we should put this on Marius shoulders now :smile:

Reply:
I think that with new files, we should add the correct URL.